pyanaconda/modules/payload/live/live_image.py

```diff
@@ -232,11 +233,15 @@ def pre_install_with_task(self):
 
     def post_install_with_task(self):
```

Comment:
Please, rename the method to `post_install_with_tasks`. The DBus API should be updated as well.

Reply:
Fixed.

Reply:
Eventually, all Payload installation methods should return a list of tasks, but this is good enough for now.